packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal.tsx

```diff
@@ -160,6 +167,11 @@ const JurisdictionModal = ({
             }
         }
     };
+
+    const flipCard = (cardName: 'svg' | 'bvi' | 'vanuatu' | 'labuan' | 'maltainvest') => {
```

Comment:
I believe we already have a proper type for this, no need to hardcode everything